### DISCUSSION TITLE
Is there a way to install a specific version of a game?

### DISCUSSION THREAD

**Discussion by dylanxyz** [legendary-gl/legendary#520] [Q&A] [answered]
I just want to install a older version of Subnautica that is compatible with the Nitrox mod. Is there any possibility?

**Comment by derrod** [comment, marked as answer]
Only if you have the manifest for that version (if you installed that older version with legendary, it'll be in `.config/legendary/manifests/old`).

**Reply by dylanxyz** [reply]
Thanks!

**Comment by marconett** [comment]
Follow up question to this: is there a way to get (or maybe reconstruct) manifests of older versions?

**Reply by derrod** [reply]
You can't get them as far as I know. The API doesn't allow you to specify older versions (or at least, not with a normal user token). Even if you know the filename of an older version you generally can't download them as manifests require authentication.

As far as reconstructing goes; The process of creating a Manifest is more or less deterministic, if you have the files of an older version you could *maybe* at least recreate the chunk data using the [BuildPatchTool](https://dev.epicgames.com/docs/epic-games-store/publishing-tools/store-presence/upload-binaries/bpt-instructions-150?sessionInvalidated=true) so that it would match what's on Epic's CDN. But the metadata would likely be different so the manifest itself would not be identical.

**Reply by marconett** [reply]
Thank you for the detailed and quick response!
